harvdev_utils/psycopg_functions/__init__.py

```diff
@@ -0,0 +1,41 @@
+from .establish_db_connection import establish_db_connection
+from .connect import connect
+from .set_up_db_reading import set_up_db_reading
+from .fb_feature_classes import Feature
+from .fb_feature_classes import Allele
+from .fb_feature_classes import Construct
+from .fb_feature_classes import Gene
+from .fb_feature_classes import Insertion
+from .fb_feature_classes import SeqFeat
+from .fb_feature_classes import Tool
+from .sql_queries import current_features
```

Comment:
Consider using
from .sql_queries import (
    current_feat_symbol_sgmls, current_feat_fullname_sgmls,
    rel_features, etc
)
To reduce the huge number of lines and can be read more easily (well i think so)

Reply:
Done